upi/openstack/inventory.yaml

```diff
@@ -5,21 +5,33 @@ all:
       ansible_python_interpreter: "{{ansible_playbook_python}}"
 
       # User-provided values
+      # same value as install-config.ymal networking.clusterNetwork.cidr
```

Comment:
clusternetwork is a list afaik.

Reply:
This is incorrect. it should be machineCIDR not clusterNetwork. Cidr

Reply:
or "machineNetwork" in newer version.